app/src/main/java/org/kiwix/kiwixmobile/di/ActivityScope.kt

```diff
@@ -0,0 +1,7 @@
+package org.kiwix.kiwixmobile.di
```

Comment:
I remember I have used `ActivityScope`. I think we call it `PerActivity`, you can reuse that and rename it if you like.

Reply:
I wanted to keep `dagger.android` and `dagger` separate was my motivation of adding this new annotation

Reply:
It will be quite confusing having two annotation sounding to provide the same functionality.

Reply:
Deleted PerActivity, on a green build I will merge this so we can do another beta release today

Reply:
When are you going to merge release/3.0 with develop? Otherwise, it will be very difficult to merge them later.

Reply:
we are going to merge the `release/3.0` branch to master and tag it for the full release, at that time I will also merge it to dev